SDWebImage/SDImageCache.m

```diff
@@ -409,14 +414,15 @@ - (NSOperation *)queryDiskCacheForKey:(NSString *)key done:(SDWebImageQueryCompl
         }
 
         @autoreleasepool {
+            NSData *diskData = [self diskImageDataBySearchingAllPathsForKey:key];
```

Comment:
the code above is not optimal, I know. I think I should change it. We explicitly call `diskImageDataBySearchingAllPathsForKey:` that loads from disk and then call `diskImageForKey:` that also calls `diskImageDataBySearchingAllPathsForKey:` and transforms that into a UIImage. Too bad we don't have tuples like in Swift, we could just change the `diskImageDataBySearchingAllPathsForKey:` to return (UIImage *, NSData *)